5EUNG/app.js

```diff
@@ -56,11 +56,144 @@ app.post('/signup', async (req, res) => {
   res.status(201).json({ message: "userCreated" });
 });
 
+app.post('/posts', async (req, res) => {
+  const { title, content, postImage, userId } = req.body
+
+  await appDataSource.query(
+    `INSERT INTO posts(
+      title,
+      content,
+      post_image,
+      user_id
+      ) VALUES (?, ?, ?, ?);
+    `,
+    [title, content, postImage, userId]
+  )
+
+  res.status(201).json({ message: "postCreated" });
+})
+
+app.get('/posts', async (req, res) => {
+  await appDataSource.query(
+    `SELECT
+      u.id AS userId,
+      u.profile_image AS userProfileImage,
+      p.id AS postingId,
+      p.post_image AS postingImageUrl,
+      p.content AS postingContent
+   FROM users u
+   INNER JOIN posts p ON u.id = p.user_id`,
+    (err, rows) => {
+      res.status(200).json({ data: rows });
+    }
+  );
+});
```

Comment:
SQL 쿼리문 indent를 맞춰주세요 !
```suggestion
app.get('/posts', async (req, res) => {
  await appDataSource.query(
    `SELECT
      u.id AS userId,
      u.profile_image AS userProfileImage,
      p.id AS postingId,
      p.post_image AS postingImageUrl,
      p.content AS postingContent
   FROM users u
   INNER JOIN posts p 
   ON u.id = p.user_id`,
    (err, rows) => {
      res.status(200).json({ data: rows });
    }
  );
});
```